eval.py

```diff
@@ -378,7 +379,7 @@ def get_stats( num_tp: Number, num_gt: Number, num_pred: Number,
     return stats
 
 def get_occluded_stats( num_tp_occluded: Number, num_gt_occluded: Number,
-                            precision: float) -> dict[str,float]:
+                       precision: float, prefix: str, include_fscore=True) -> dict[str,float]:
```

Comment:
I'm not sure why `include_fscore` needs to be a parameter. Is there any reason you wouldn't want to just include it in the return value, since it will be calculated? That would clear. up other parts of the code perhaps.

In any case, the additional parameter(s) need to be added to the documentation block, and the description of the keys also needs to be updated, since it is no longer fixed.

Reply:
I dont want to calculate the f1 score for the occluded subcategories. Is there a better way to do it without modifying the function arguments?

Reply:
As written, it's already *calculated*; the conditional merely excludes its return. Since the function is written (and documented) to produce R&F (stats, plural), the conditional behavior is a bit odd. 

I see two ways to proceed:

1. Calculate and return the values; little harm in doing so, and the RRC front-end does not need to display every metric calculated and returned. (My preference.) Then you don't need to mess about with all the cases  elsewhere that have to specify the extra argument.
2. Move the conditional so that F-score is not even calculated, let alone returned, and also update the documentation for the return value to reflect this change.